components/helper/ShareButtons.tsx

```diff
@@ -9,9 +12,131 @@ import {
   TwitterShareButton,
 } from 'react-share';
 
-export default function ShareButtons() {
+type ShareButtonsProps = {
+  popup: boolean;
+  setCopied: React.Dispatch<React.SetStateAction<boolean>>;
+  copied: boolean;
+};
+
+export default function ShareButtons({
+  popup, setCopied, copied,
+}: ShareButtonsProps) {
   const url = 'https://clearviction.org/calculator/head-initial-1-cont';
 
+  const style = {
+    fontSize: '1.25rem',
+    fontWeight: 400,
+    lineHeight: '1.375rem',
+    letterSpacing: '.0156rem',
+    textAlign: 'left',
+    color: '#2F3554',
+    textDecoration: 'underline',
+    marginLeft: '.4375rem',
+  };
+
+  const handleCopyClick = async () => {
+    await navigator.clipboard.writeText(url);
+    setCopied(true);
+  };
+
+  function handleEmailClick() {
+    const body = encodeURIComponent(url);
+    const mailtoUrl = `mailto:?body=${body}`;
+    window.open(mailtoUrl);
+  }
+
+  if (copied) {
+    return (
+      <Box sx={{
+        width: '20.3125rem',
+        height: '5.9375rem',
+        padding: '1rem',
+        borderRadius: '.375rem',
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+      }}
+      >
+        <Typography
+          variant="h5"
+          sx={{
+            fontFamily: 'Sintony',
+            fontWeight: 600,
+            textAlign: 'center',
+            marginBottom: '0',
+            fontSize: '1.5rem',
+
+          }}
+        >
+          Link Copied!
+
+        </Typography>
+      </Box>
+    );
+  }
+
+  if (popup) {
+    return (
+      <Box sx={{
```

Comment:
It would be helpful here to have some CSS selectors to identify what these are. When we use a lot of unidentified boxes, it makes it harder to debug or to understand.